pkgs/os-specific/linux/systemd/0019-Revert-pkg-config-prefix-is-not-really-configurable-.patch

```diff
@@ -1,4 +1,4 @@
-From cd5b1075499b8498d9c700a317ad11a3199c447a Mon Sep 17 00:00:00 2001
+From ba59e81758c3ae0382ee491181869ae4338a0f22 Mon Sep 17 00:00:00 2001
```

Comment:
This one is now an empty diff?

Reply:
No. It still sets prefix.

Reply:
I just double checked the .pc files. They are still correct.